apl-core/src/main/java/com/apollocurrency/aplwallet/apl/eth/service/EthereumWalletService.java

```diff
@@ -88,6 +88,18 @@ public EthWalletBalanceInfo balanceInfo(String address){
         return ethWalletBalanceInfo;
     }
 
+    
+     /**
+     * Get Eth balance for both PAX-ETH wallets
```

Comment:
description is not matching to method's name.

Reply:
all necessary alterations have been made. Regard it as done )